app/root.tsx

```diff
@@ -235,10 +235,11 @@ function UserDropdown() {
 					</DropdownMenu.Item>
 					<DropdownMenu.Item asChild>
 						<Form
-							action="/logout"
-							method="POST"
 							className="rounded-b-3xl px-7 py-5 outline-none radix-highlighted:bg-night-500"
-							onClick={e => submit(e.currentTarget)}
+							onClick={e => {
+								e.preventDefault()
```

Comment:
Is there a reason this is not the only line changed in the PR? Why can't we keep the `action` and `method` and use `submit(e.currentTarget)`?

Reply:
Let me try that out

Reply:
I'm not sure how the Radix UI internals work, but my suspicion is that, because the form takes up all of the `<DropdownMenu.Item asChild>`'s behavior, having an onClick on it is already too late, because it'll send the close comand to the `<DropdownMenu.Portal>`.

Adding `onClick={e => e.stopPropagation()}` to the button works 👌